lens/base/view-set.rkt

```diff
@@ -6,10 +6,11 @@
 (module+ test
   (require rackunit))
 
-(provide lens-view
-         lens-set
-         lens-view*
-         lens-set*)
+(provide
+ lens-view*
+ lens-set*
```

Comment:
For `lens-view*`, it's easy:
```racket
(->* [any/c] #:rest (listof lens?) any/c)
```
And for both `lens-set*` and `lens-transform*`, perhaps a special contract combinator that produces a contract equivalent to
```racket
(define (combinator first-val/c second-val/c)
  (define c
    (cons/c first-val/c (cons/c second-val/c (recursive-contract c))))
  c)
```
And you could use it for both `lens-set*` and `lens-transform*`.